packages/eslint-rules/lib/rules/no-custom-color.js

```diff
@@ -9,6 +9,7 @@ const alwaysValidClasses = [
   'text-lg',
   'text-xl',
   'text-ellipsis',
+  'bg-clip-text',
```

Comment:
Where do we use this one?

Reply:
On the title, to get the gradient effect.
https://github.com/dailydotdev/apps/blob/fae17e358f191ec7360cfe205ea1185bd8c884d4/packages/webapp/pages/onboarding.tsx#L64-L67